docs/examples/user-configuration/user-config-template.yaml

```diff
@@ -1,145 +1,212 @@
-apiVersion: template.openshift.io/v1
+apiVersion: v1
 kind: Template
 labels:
-  template: user-config
-message: A new data grid service has been created in your project.
+  template: datagrid-service
 metadata:
   annotations:
-    description: An example Red Hat JBoss Data Grid application. For more information
-      about using this template, see https://github.com/jboss-openshift/application-templates.
+    description: Red Hat Data Grid is an in-memory, distributed key/value store.
     iconClass: icon-datagrid
-    openshift.io/display-name: Red Hat JBoss Data Grid 7.2 (User Config)
+    openshift.io/display-name: Red Hat Data Grid Service
+    openshift.io/documentation-url: https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.3/html/red_hat_data_grid_for_openshift/
+    openshift.io/long-description: Red Hat Data Grid Service provides a fully customizable implementation of Data Grid for OpenShift that is stable, persistent, and elastically scalable.
     openshift.io/provider-display-name: Red Hat, Inc.
-    tags: datagrid,jboss
-    template.openshift.io/documentation-url: https://access.redhat.com/documentation/en/red-hat-jboss-data-grid/
-    template.openshift.io/long-description: This template defines resources needed
-      to develop Red Hat JBoss Data Grid 7.2 based applications, including a deployment
-      configuration, using ephemeral (temporary) storage and communication using http.
-    template.openshift.io/support-url: https://access.redhat.com
-    version: "1.2"
+    openshift.io/support-url: https://www.redhat.com/en/services/support
+    tags: database,datagrid
+  labels:
+    template: datagrid-service
```

Comment:
Labels and descriptions should not be changed IMO other than `7.2 to `7.3` where required and links to appropriate docs.